src/west/configuration.py

```diff
@@ -130,6 +130,19 @@ def delete(self, option: str):
             self.cp.write(f)
 
 
+class _Converter:
+    @staticmethod
+    def str_to_paths(paths: str | None, sep: str = os.pathsep) -> list[Path]:
+        """Split a string into a list of Path objects using the given separator."""
```

Comment:
I'm concerned pydocs automatically make these part of the API, is that correct? They're too basic and unrelated to the API to be part of it.